Controllers/questionController.js

```
@@ -22,20 +29,27 @@ export const questionController = {
                     errors: validation.errors
                 });
             }
+            console.info(`Published question_create event with messageId: ${messageId}`);
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)